src/Paket/Program.fs

```diff
@@ -49,6 +49,9 @@ type CLIArguments =
     | [<AltCommandLine("-f")>] Force
     | Hard
     | [<CustomCommandLine("nuget")>] Nuget of string
+//    | [<CustomCommandLine("gist")>] Gist of string * string
+//    | [<CustomCommandLine("github")>] Github of string * string
+    | [<CustomCommandLine("url")>] Url of string * string
     | [<CustomCommandLine("version")>] Version of string
     | [<CustomCommandLine("add-credentials")>] AddCredentials of string
```

Comment:
I'm not sure, but maybe it will be easier after https://github.com/fsprojects/Paket/pull/551 - can you try this?